### PR TITLE
Better error display on SV inspector/spreadsheet view on import form

### PR DESCRIPTION
**Pull request by cmdcolin**
Possibly fix for https://github.com/GMOD/jbrowse-components/issues/3219

### CODE REVIEW

File: plugins/spreadsheet-view/src/SpreadsheetView/components/ImportWizard.js [hunk 1]
```diff
@@ -52,6 +51,7 @@ const NumberEditor = observer(
 )
 
 const ErrorDisplay = observer(({ error }) => {
+  console.log({ error })
```
Comment (carolinebridge-oicr) [inline plugins/spreadsheet-view/src/SpreadsheetView/components/ImportWizard.js:54]: console log may have been left over here

Comment (cmdcolin) [reply plugins/spreadsheet-view/src/SpreadsheetView/components/ImportWizard.js]: +1 that entire component was replaced instead with usage of @jbrowse/core/ui/ErrorMessage. also added some typescripting to that file

**Comment by carolinebridge-oicr** [comment]
Not sure if this is a me-thing and my display or something but could you increase the default height of the view for both of these such that the open button is visible from the get-go? If I am scrolled down to click the Open button for the file, I actually can't see the error message because it renders at the top of the view and I'm scrolled to the bottom.

Not sure if this communicates what I mean, but here's some ss's:
Screen Shot 2022-09-26 at 3 12 08 PM
^ default when the view opens the Open button is obscured below the scroll
Screen Shot 2022-09-26 at 3 12 16 PM
^ scrolling down its there but the top gets cut off, the error message is actually displayed in this ss but obscured by the scroll

**Comment by cmdcolin** [comment]
I have same thing on my computer with scrolling down to open button, etc. I'll see if I can adjust

**Pull request merged**